centreon/tests/e2e/features/Cloud-notifications/04-notification-edit.feature

```diff
@@ -0,0 +1,45 @@
+@ignore
+@REQ_MON-24182
+Feature: Editing notification rule configuration
+  As a Centreon user with access to the notification rules page
+  I need to edit a notification rule
+  So that I can adapt to to the ever evolving needs of my monitoring system
+
+  Background:
+    Given a user with access to the notification rules page
+    And a notification rule is already created
+    And the user is on the notification rules page
+
+  Scenario: Editing a notification rule resources configuration
+    When the user selects the edition action on a notification rule
+    And the user changes the resources selection and corresponding status changes parameters
+    And the user saves and confirms
```

Comment:
do save and confims behaviors both happen at the same time when clicking the save button ?

Reply:
it's only one button, to confirm